.idea/kotlinc.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
No. I guess because I have the Kotlin plugin installed it's adding this file to every project. Should I git-ignore it?